recipe/meta.yaml

```diff
@@ -23,13 +23,13 @@ requirements:
     - cython
   run:
     - python
-    - async-timeout <4.0,>=3.0
+    - async-timeout >=4.0a2,<5.0
     - attrs >=17.3.0
     - chardet >=2.0,<4.0
     - idna_ssl >=1.0  # [py<37]
-    - multidict >=4.5,<5.0
+    - multidict >=4.5,<7.0
     - typing-extensions >=3.6.5  # [py<37]
-    - yarl >=1.0,<1.6.0
+    - yarl >=1.0,<2.0
```

Comment:
This is for master. 3.6.3 is what we have here. See https://github.com/aio-libs/aiohttp/blob/19405036da709906b3349298569c4ae8696d436b/setup.py#L75-L81